tools/image_processing/woundhealing/woundhealing.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="woundhealing" name="Wound healing scratch assay image analysis" version="@VERSION@+galaxy1">
+    <description>Wound healing scratch assay image analysis</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive">
+    <![CDATA[
+	bash '$script_file' &&
+        ImageJ --ij2 --headless --run '$__tool_directory__/measureWoundClosing.groovy' 'inputDir="./input",datasetId="$dataset_id",threshold="$threshold",headless="true",saveResults="true",outDirName="./output"'
+    ]]>
+    </command>
+    <configfiles>
+        <configfile name="script_file">
+            mkdir ./input
+
+            #if $con_input_type.input_type =="yes"
+                tar -C ./input -xvf $con_input_type.input_images
+            #else
+                #for $i, $filename in enumerate($con_input_type.input_images):
+                    ln -s '$filename' './input/${filename.element_identifier}'
+                #end for
+            #end if
+        </configfile>
+    </configfiles>
+
+    <inputs>
+        <conditional name="con_input_type">
+            <param name="input_type" type="select" label="Are the input images packed into a tar archive?">
+                <option value="yes">Yes</option>
+                <option value="no">No</option>
+            </param> 
+            <when value="no">
+                <param name="input_images" type="data" multiple="true" format="@FORMATS@" label="Images" />
+            </when>
+            <when value="yes">
+                <param name="input_images" type="data" format="tar" label="A tarball of images" />
+            </when>
+        </conditional>
+        <param name="dataset_id" type="text" label="Dataset ID" />
+        <param name="threshold"  type="text" label="CoV threshold (-1: auto)" value="-1"/>
+    </inputs>
+    <outputs>
+        <data name="tif_output" format="tif" from_work_dir="output/*.tif" label="movie output ${on_string}" />
+	<data name="csv_output" format="tabular" from_work_dir="output/*.csv" label="tabular output"/>
```

Comment:
I prefer this to be a tabular - tab separated - because galaxy has more tools to deal with those.
But the file that is created is a comma-seperated file, isn't it?

You could also add the columns names as metadata: https://github.com/galaxyproject/tools-iuc/blob/3a3dfd676076438bea1518eb731790edfb4da3c0/tools/onto_toolkit/get_term_id_vs_term_name.xml#L16

Reply:
- yes, the output file created by fiji is a comma-separated file.

- I have tried the 'location' tag on `tests->test->param` . planemo lint told me that the `location `tag is not allowed, however, the xml ref seems to allow it. My planemo is up to date, v0.75.14. not sure if it's to do with the ci update

- I also have question about multiple urls as value of the location tag. If the input `multiple=true`, and in the test, I assume I will need to separate multiple urls by comma? The only example I can find its the `mzmine_batch` tool in iuc repo, but they are commented out.
 
thanks, I will work on all your suggestions above, and come back later.

Reply:
You really need to have the latest dependencies of planemo. The location attributes are only in release_23.1.
Try updating your galaxy-tool-utils package.

I assume its comma separated. But haven't tried myself yet.

Reply:
thanks.. i tried the update of galaxy-toolutils to 23.1.1, but the latest planemo 0.75.17 and the dev 0.75.18.dev0 does not support it yet.. i guess we simply need to wait a few more days, in this case?

Reply:
Yeah, then we need to wait a bit.